src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -124,6 +126,7 @@ private static bool TryGetAttributeAdditionalErrors(Dictionary<string, OpsMetada
                 { "MicrosoftAlias", new string[] { "ms-alias-invalid" } },
                 { "Deprecated", new string[] { "attribute-deprecated" } },
                 { "Uniqueness", new string[] { "duplicate-attribute" } },
+                { "Length", new string[] { "description-bad-length" } },
```

Comment:
Map `Length` to a general error code that could apply to any attribute, so we can reuse it for checking length for other metadatas.

```suggestion
                { "Length", new string[] { "string-length-invalid" } },
```

Reply:
ok. yml test case added.